corehq/apps/ota/views.py

```diff
@@ -139,21 +139,31 @@ def claim(request, domain):
     as_user_obj = CouchUser.get_by_username(as_user) if as_user else None
     restore_user = get_restore_user(domain, request.couch_user, as_user_obj)
 
-    case_id = unquote(request.POST.get('case_id', ''))
-    if not case_id:
+    # This will be expecting space seperated values of all the cases to be claimed
+    case_ids = request.POST.getlist('case_id')
+
+    if not case_ids:
         return HttpResponse('A case_id is required', status=400)
 
     try:
-        if get_first_claim(domain, restore_user.user_id, case_id):
-            return HttpResponse(status=204)
+        case_ids_already_claimed = get_first_claims(domain, restore_user.user_id, case_ids)
+        if case_ids_already_claimed:
+            case_ids_to_claim = [case for case in case_ids if case not in case_ids_already_claimed]
+        else:
+            case_ids_to_claim = case_ids
+    except CaseNotFound as err:
+        return HttpResponse(f'No cases claimed. Case IDs "{err}" not found',
```

Comment:
How does this string formatting work?  Isn't `err` an exception instance?

Reply:
In this case it prints out the ids that were not found since I changed this function:
```python3
def get_first_claims(domain, user_id, case_ids):
     ...

    if len(cases_not_found) != 0:
        raise CaseNotFound(", ".join(cases_not_found))

     ...
```